.github/pull_request_template.md

```diff
@@ -4,5 +4,6 @@
 
 ## Contribution Guidelines
 
-- [ ] By creating this pull request, I acknowledge that I have read the [Contribution Guidelines](https://github.com/material-extensions/vscode-material-icon-theme/blob/main/CONTRIBUTING.md) for this project.
+- [ ] By creating this pull request, I acknowledge that I have read the [Contribution Guidelines](https://github.com/material-extensions/vscode-material-icon-theme/blob/main/CONTRIBUTING.md) for this project
+- [ ] The PR title follows the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/)
```

Comment:
I don't think it's worth adding, because it takes time to confirm the fields, and it will take a long time, it's enough that the contributor read `CONTRIBUTING.md`

```suggestion
```

Reply:
you're right. I wasn't sure about it.